### ISSUE TITLE
Wht is cn gpu? 

### ISSUE THREAD

**Issue by rb116** [fireice-uk/xmr-stak#2218]
Hi may I know the difference between cn gpu and xmr stak exe? I m currently on 2.7 , mining cn v2/8. Shd I upgrade to 2.8? Bcos u mentioned it will be more power hungry ?

**Comment by Spudz76** [comment]
nah, unless you mine RYO, which is the only coin currently using that hashtype (CN-GPU is a new algo)

Other currency types are still good old whatever algo they were like v8 (cn2v2) with no changes in power.  

There were some changes in AMD driver compatibility, some bugfixes in nVidia too, so it is worth upgrading (once released).

**Comment by Ryo-RU** [comment]
rb116, it is new mining algo.
if you choose "ryo" in miner it will auto switch on fork. if you want to force new algo in testnet pool of ryo before actual fork or in mining other coins on this pow (in future) you can use "cryptonight_gpu"
More info - here: [https://www.reddit.com/r/ryocurrency/comments/an11cy/updatem_ryo_currency_0400_lorentz_transform/](https://www.reddit.com/r/ryocurrency/comments/an11cy/updatem_ryo_currency_0400_lorentz_transform/)

**Comment by rb116** [comment]
Thanks for the enlighten. 


Sent from Yahoo Mail for iPhone


On Saturday, February 9, 2019, 1:17 PM, Tony Butler <notifications@github.com> wrote:


nah, unless you mine RYO, which is the only coin currently using that hashtype (CN-GPU is a new algo)

Other currency types are still good old whatever algo they were like v8 (cn2v2) with no changes in power.

There were some changes in AMD driver compatibility, some bugfixes in nVidia too, so it is worth upgrading (once released).
